src/Stratis.Bitcoin/Connection/ConnectionManagerBehavior.cs

```diff
@@ -51,44 +50,26 @@ protected override void AttachCore()
         {
             this.logger.LogTrace("()");
 
-            this.AttachedNode.StateChanged += AttachedNode_StateChanged;
-            this.AttachedNode.MessageReceived += AttachedNode_MessageReceived;
+            this.AttachedNode.StateChanged += this.AttachedNode_StateChanged;
             this.chainHeadersBehavior = this.AttachedNode.Behaviors.Find<ChainHeadersBehavior>();
 
             this.logger.LogTrace("(-)");
         }
 
-        private void AttachedNode_MessageReceived(Node node, IncomingMessage message)
-        {
-            this.logger.LogTrace("({0}:'{1}',{2}:'{3}')", nameof(node), node.RemoteSocketEndpoint, nameof(message), message.Message.Command);
-
-            if (this.chainHeadersBehavior.InvalidHeaderReceived && !this.Whitelisted)
-            {
-                node.DisconnectAsync("Invalid block received");
-            }
-
-            this.logger.LogTrace("(-)");
-        }
-
         private void AttachedNode_StateChanged(Node node, NodeState oldState)
         {
             this.logger.LogTrace("({0}:'{1}',{2}:{3},{4}:{5})", nameof(node), node.RemoteSocketEndpoint, nameof(oldState), oldState, nameof(node.State), node.State);
 
             if (node.State == NodeState.HandShaked)
             {
                 this.ConnectionManager.AddConnectedNode(node);
-                this.infoLogger.LogInformation("Node {0} connected ({1}), agent {2}, height {3}", node.RemoteSocketEndpoint, this.Inbound ? "inbound" : "outbound", node.PeerVersion.UserAgent, node.PeerVersion.StartHeight);
+                this.infoLogger.LogInformation("Node '{0}' connected ('{1}'), agent '{2}', height '{3}'", node.RemoteSocketEndpoint, this.Inbound ? "inbound" : "outbound", node.PeerVersion.UserAgent, node.PeerVersion.StartHeight);
```

Comment:
this won't be as pretty on the console, esp. `connected ('inbound')` part. also height is a number, does not really belong between apostrophes.

Reply:
Ah I did think this is following your pattern where you add`'{0}'` to all trace outputs.
Sure will fix

Reply:
actually not to all of them. this is INFO level, so not tracing at all. optimally (imho) we would not have bound logging with console output, so in that case i would use the notation everywhere, but as we have INFO+ on console, one should rather prefer nice console output instead of following the notation

however, the notation is described here - https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/logging-style.md#value-log-format - this says where to use apostrophes and where not